config/peerpods/podvm/azure-podvm-image-handler.sh

```diff
@@ -666,7 +666,7 @@ function delete_image_using_id() {
     [[ -z "${IMAGE_ID}" ]] && error_exit "IMAGE_ID is empty"
 
     # Delete the image
-    az image delete --ids "${IMAGE_ID}" ||
+    az image-version delete --ids "${IMAGE_ID}" ||
```

Comment:
Providing some reference URLs to clarify this change would be appreciated (but not strongly required).

Reply:
Looking at your comment made me realise a typo. The command should be `az sig image-version delete ..`. I have fixed it.